### ISSUE TITLE
Future Plans

### ISSUE THREAD

**Issue by safak45xx** [safak45xx/Blurfox-MacOS-#2]
![Ekran Resmi 2024-02-19 18 15 20](https://github.com/safak45xx/Blurfox-MacOS-/assets/141409983/366ed507-cae4-4c73-a40b-5fcc2e394542)
I don't exactly know how to do this, but if I succeed, a few of the first things I'll add.

**Comment by safak45xx** [comment]
The code I'm writing right now is quite messy. In future versions, I will organize everything in a more understandable way.

**Comment by plyght** [comment]
yes, is there any way to change [this](https://github.com/safak45xx/Blurfox/assets/81438024/28f241a0-5ca8-4fe8-85ff-11e2bea5f560) to make it bigger?
